pallets/nft/src/tests.rs

```diff
@@ -66,13 +70,13 @@ fn test_issue_and_claim_eth() {
 			ClassType::Claim(merkle_root),
 		));
 
-		assert_eq!(
-			events(),
-			[Event::Nft(crate::Event::CreatedClass(
-				alice_account.clone(),
-				0
-			)),]
-		);
+		// assert_eq!(
```

Comment:
@Satoshi-Kusumoto Let's convert this to an issue to fix, or just drop it.

Reply:
You just reminded me about that
Thx